packages/typespec-python/test/mock_api_tests/test_azure_client_generator_core_internal.py

```diff
@@ -0,0 +1,40 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+# import pytest
+# from _specs_.azure.clientgenerator.core.internal import InternalClient
+# from _specs_.azure.clientgenerator.core.internal import models
+# from _specs_.azure.clientgenerator.core.internal.models import _models
+
+
```

Comment:
@tadelesh it started failing with not being able to access `_models` inside the `models` folder. can you investigate and add tests? thanks! sorry for leaving this on your plate

Reply:
For current `internal` test, there is no exported model, so the import need to change to `from ..models import InternalModel`. Since it will not happen in the real spec and we also want to test different case for internal transitive closure, I decide to change the cadl ranch test case.